Add a few Readme comments (#53)
makerdao/lockstake: README.md

For the mentioned examples of `chop` and `fee` we get:
`mat > 1.13 / 0.85 ~= 133%`

Note that in practice the `mat` value is expected to be significantly larger and have buffers over this rough calculation.

> It should take into account market fluctuations and protocol safety, especially for using the governance token as collateral.

Reviewer: ```suggestion
It should take into account market fluctuations and protocol safety, especially considering that the governance token is used as collateral.
```
